lib/rules/no-async-actions.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const utils = require('../utils/utils');
+
+
+//------------------------------------------------------------------------------
+// General rule - Don't use async actions
+//------------------------------------------------------------------------------
+
+
+const message = 'Do not use async actions';
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Disallow usage of async actions in components',
+      category: 'Possible Errors',
+      recommended: true,
```

Comment:
Rules should never be added as recommended since that would be a breaking change. We can consider making it recommended in the future.

Reply:
Removed the recommended flag